scalableminds/webknossos
# Register new demo instance users with mailchimp

Do not send welcome mail to new demo instance users, instead add them to mailchimp list and tag them either with `registered_as_admin` or `registered_as_user`

### Steps to test:
- set isDemoInstance = true, fill in mail.mailchimp block in config with valid values
- register new account, mailchimp list should contain new entry

### Issues:
- fixes #5424
